Karamelfile

```diff
@@ -27,3 +27,13 @@ dependencies:
   - recipe: ndb::_test
     global:  
       - ndb::mysqld
+  - recipe: ndb::ndb_restore_meta
+    global:
+      - ndb::ndbd
```

Comment:
shouldn't this be dependent on mysqld recipe?

Reply:
No, I restore the schema with `ndb_restore` program which doesn't use mysqld